### DISCUSSION TITLE
When the antivirus becomes the virus – Let’s Do Digital

### DISCUSSION THREAD

**Discussion by giscus** [letsdodigital/letsdodigital.github.io#21] [General]
# When the antivirus becomes the virus – Let’s Do Digital

And how we should manage digital health deployment

https://letsdodigital.org/posts/2024-07-19-when-the-anti-virus-becomes-the-virus/

**Comment by BB-1313** [comment]
Given this was prompted at the DH summer schools and some of the delegates held a D&D session why not fully commit, channel the inner gamer and go for "skill trees"!?
 
The pyramid is a solid foundation in response to the important and unanswered question of what are the priorities and pre-requisites but it may not reflect the (beautiful) complexity and interconnectivity of a truly functional system. As opposed to the patchwork we have so far.
 
Perhaps the different tracks might include:
* Data/information
* Staff and human resources
* Physical and technical infrastructure
* Clinical knowledge including the interaction with the above
* Regulatory/legal/policy/strategy
 
The relative heirarchies within and between woudl also need to be defined but would basically be represented by the components and connections in the "tree".
 
Top level out puts (AI, "systems" and functions) would sit on top rather than be the focus as is the status quo.
